### DISCUSSION TITLE
kpanda/user-guide/services-routes/create-services/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3010] [Announcements]
# kpanda/user-guide/services-routes/create-services/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/services-routes/create-services/

**Comment by LLLzxx** [comment]
缺少 创建“外部服务” 访问类型的介绍

**Reply by lemontrees123** [reply]
已更新：https://docs.daocloud.io/kpanda/user-guide/network/create-services/#_2
